src/main/webapp/app/exercise/exercise-action-bar/exercise-action-bar.component.html

```
@@ -0,0 +1,105 @@
+<div #actionsRow class="d-flex gap-1 align-items-center justify-content-end flex-nowrap w-100">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)